OloEditor/src/EditorLayer.cpp

```
@@ -1206,6 +1247,109 @@ namespace OloEngine
         Application::Get().GetWindow().SetTitle(title);
     }
 
+    bool EditorLayer::TerrainRaycast(const glm::vec2& mousePos, const glm::vec2& viewportSize, glm::vec3& outHitPos) const
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)